### PR TITLE
Improve documentation on extensions

### PR DESCRIPTION
**Pull request by Rensvandeschoot**
This PR improves the documentation on extensions and adds a list of extensions developed by users.

### CODE REVIEW

File: docs/source/extensions/overview_extensions.rst [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+- :doc:`extension_wordcloud`: Create a visual impression of the contents of datasets via a wordcloud.
+- ``asreview-statistics``: Tool to give some basic properties of a dataset, such as number of
+  papers, number of inclusions. `GitHub <https://github.com/asreview/asreview-statistics>`__
+- ``asreview-hyperopt``: Optimize the hyperparameters of the models in ASReview. `GitHub <https://github.com/asreview/asreview-hyperopt>`__
+
+
+
+.. _extensions-users:
+
+Extensions Developed by Users
```
Comment (J535D165) [inline docs/source/extensions/overview_extensions.rst:34]:
```suggestion
Community-maintained extensions
```

I prefer to use "community-maintained", as it is more clearly a third party is maintaining it. 
See also https://help.ubuntu.com/community/Repositories/Ubuntu

File: docs/source/extensions/overview_extensions.rst [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+-----------------------------
+
+The following extensions are developed by users:
```
Comment (J535D165) [inline docs/source/extensions/overview_extensions.rst:37]:
```suggestion
The following extensions are developed and maintained by the ASReview community:
```

File: docs/source/extensions/overview_extensions.rst [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+- XXX
```
Comment (J535D165) [inline docs/source/extensions/overview_extensions.rst:39]:
```suggestion
Work in Progress
```

**Review by J535D165 — CHANGES_REQUESTED** [review]
Thanks!

This PR will break a lot of links in the Documentation, READMEs, other repos, and website. What's the plan regarding this?

File: docs/source/extensions/overview_extensions.rst (hunk 1 above)
Comment (Rensvandeschoot) [reply docs/source/extensions/overview_extensions.rst]: I agree, nice suggestion

**Review by J535D165 — APPROVED** [review]

**Comment by Rensvandeschoot** [comment]
@J535D165 thank you for your review! 

Broken links in the documentation are checked with linkchecker. For the website I have a broken link detector installed. For other repositories we have to manually check the broken links. Let me know if you are publishing a new release and I will make time to go trough all other repositories.

**Pull request merged**